roles/grafana_cloud_operator/tasks/delete_grafana_oncall_hub_spoke.yml

```diff
@@ -0,0 +1,156 @@
+- name: Debug at the start of hub spoke
```

Comment:
Do we still need this debug step? 
If yes, you can modify the msg to "Started processing deletion of hub spoke tasks"